dali/operators/decoder/image_decoder.cc

```diff
@@ -227,11 +227,9 @@ anchors and shapes.
 
 Inputs must be supplied as tensors in the following order:
 
-* ``data`` that contains the input data.
-* ``anchor`` that contains normalized or absolute coordinates, depending on the
-  ``normalized_anchor`` value, for the starting point of the slice (x0, x1, x2, and so on),
-* ``shape`` that contains normalized or absolute coordinates, depending on the
-  ``normalized_shape`` value, for the dimensions of the slice (s0, s1, s2, and so on).
+#. ``data``
+#. ``anchor``
+#. ``shape``
```

Comment:
Do we still need that if InputDox is provided?

Reply:
It has some value, it tells you the order of those. I had my doubts. Do you think we should remove this completely?

Reply:
Just a suggestion.